quarkus-opentelemetry-exporter-gcp/deployment/src/main/java/io/quarkiverse/opentelemetry/exporter/gcp/deployment/GcpExporterAlwaysEnabledProcessor.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.opentelemetry.exporter.gcp.deployment;
+
+import io.quarkus.deployment.Feature;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+// Executed even if the extension is disabled, see https://github.com/quarkusio/quarkus/pull/26966/
+public class GcpExporterAlwaysEnabledProcessor {
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(Feature.OPENTELEMETRY_JAEGER_EXPORTER);
```

Comment:
We need a new specific feature for this new extension... let me think about this.

Reply:
Please remove this one. You don't need it.